### ISSUE TITLE
Turtle and Hare PIVX Newsletter

### ISSUE THREAD

**Issue by wdmason** [wdmason/Transcription-Captioning-Proofreading-Copywriting#15]
First draft in 48 hours if it is to make the deadline for idea submission COB Sunday.

-------
William Mason [01:20] 
I would have to go over previous newsletters and really do an in depth study of the material that is currently out there, but I've been thinking about the factors that present challenges to PIVX really taking off. Exchange exposure, debit card integration, organic growth versus ICO splash, etc. I would like to see a newsletter that gives a very powerful and very simple infograph highlighting the history of PIVX and letting the readers grasp in simplest terms why PIVX is such a powerful platform.
Pinned
Randal Quiade
So, for the next newsletter if anyone comes across some good material be sure to mention it in this chat. 

Let's aim to push it out around the middle of next week. That means completed and prepping for it's initial set of reviews by EOD Sunday.
Posted in #newsletterOct 17th at 23:05

[01:24] 
We're really not a flash-in-the-pan. It's much more a matter of slow and steady wins the race. Maybe a Tortoise and Hare-themed discussion outlining pros and cons of how PIVX is doing things versus how Flash and Splash ICOs that have proven to be without substance have fallen by the wayside. Just be sensitive not to disparage all ICOs in general. The majority are shitcoins and there's no denying that, but ICOs are a beautiful combination of IPO, Crowd Funding, and Venture Capital magnets that are perpetuating the single most significant transfer of wealth in this century.

[01:26] 
Anyway TL:DR Tortoise and Hare infograph laying it out in simple terms. That's my suggestion for next week's newsletter. Make our weaknesses our strengths and our strengths all the more powerful.

-------
